doc/onboarding.md

```diff
@@ -69,9 +75,11 @@ onboarding session.
       * When adding a semver label, add a comment explaining why you're adding it. Do it right away so you don't forget!
 
   * [**See "Who to CC in issues"**](./onboarding-extras.md#who-to-cc-in-issues)
-    * will also come more naturally over time
+    * This will come more naturally over time
+    * For many of the teams listed there, you can ask to be added if you are interested
```

Comment:
You can’t see that page if you’re not a member, so it’s at least going to be confusing for those who read through the doc a bit in advance.